source/math/FloatRandomGenerator.ooc

```diff
@@ -65,12 +75,24 @@ FloatGaussianRandomGenerator: class extends FloatRandomGenerator {
 		this _backend free()
 		super()
 	}
+	setRange: func (=_mu, =_sigma) {
+		_hasSecond = false
```

Comment:
I used setRange (noticed I've been inconsistent with naming here, will fix) for the uniform generators to reuse code, thought I'd include them in the gaussian versions for similarity. Possibly unneccessary.